spec/features/add_photo_to_entry_spec.rb

```diff
@@ -31,7 +31,7 @@
     user = entry.user
     sign_in(user)
     click_link entry.title
-    click_button 'Add Photo'
+    find("#add-photo",:visible=>false).click
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space missing after comma.<br>Surrounding space missing for operator `=>`.